feature/xhr/aggregate/index.js

```diff
@@ -168,3 +194,17 @@ function onEventsHarvestFinished(result) {
     sentAjaxEvents = []
   }
 }
+
+function exceedsSizeLimit(payload, maxPayloadSize) {
+  maxPayloadSize = maxPayloadSize || MAX_PAYLOAD_SIZE
+  return window.Blob ? new Blob([payload]).size > maxPayloadSize : false
```

Comment:
What is the advantage of using Blob instead of just `payload.length`?

Reply:
Are there ever going to be cases where characters will be encoded in different ways?

Converting to a blob removes all the unknowns of encoding, special characters, or variable byte sizes from the string, but if we are never going to face those we could just assume the characters are 2 bytes per character i suppose